app/schemas/chat.rocket.android.db.RCDatabase/16.json

```diff
@@ -0,0 +1,1223 @@
+{
```

Comment:
Did you intend to include these schema files?  I'm personally unclear on whether we need to do that.

Reply:
I changed the schema of the database (Added a table to store the unsent upload files)
I need to do a small change to the schema to fix something, will generate a single file instead of two.